src/esm_runscripts/filelists.py

```diff
@@ -940,6 +940,18 @@ def copy_files(config, filetypes, source, target):
     successful_files = []
     missing_files = {}
 
+    # Initialization for parallelization with futures
+    futures = {}
+    if config["general"].get("parallel_file_movements", False) == "threads":
```

Comment:
Are the allowed options for `parallel_file_movements` documented somewhere?

Reply:
Just documented the new variable here: https://esm-tools.readthedocs.io/en/feat-use_threads_for_file_movement/esm_variables.html#runtime-variables (`parallel_file_movements`)

Note that `threads` is now the new default (defined in `configs/defaults/general.yaml`).